instrumentation/net/http/otelhttp/internal/semconvutil/httpconv.go

```diff
@@ -211,12 +211,30 @@ func (c *httpConv) ClientResponse(resp *http.Response) []attribute.KeyValue {
 	return attrs
 }
 
-// ClientRequest returns attributes for an HTTP request made by a client. The
-// following attributes are always returned: "http.url", "http.flavor",
-// "http.method", "net.peer.name". The following attributes are returned if the
-// related values are defined in req: "net.peer.port", "http.user_agent",
-// "http.request_content_length", "enduser.id".
+// TraceClientRequest returns attributes for an HTTP request made by a client. The
+// following attributes are always returned: "http.url", "http.method",
+// "net.peer.name". The following attributes are returned if the related values
+// are defined in req: "net.peer.port", "http.user_agent",
+// "http.request_content_length", "user_agent.original".
 func (c *httpConv) ClientRequest(req *http.Request) []attribute.KeyValue {
+	/* The following semantic conventions are returned if present:
+	http.method						string	Required
+	user_agent.original				string	Recommended
+	http.url						string	Required
+	net.peer.name					string	Required
+	net.peer.port					int		Conditionally Required: [5]
+	http.request_content_length		int		Recommended
+	*/
+
+	/* The following semantic conventions are not returned:
```

Comment:
What about `net.protocol.name` and `net.protocol.version`?

Reply:
For the client, the default value here is HTTP, 1.0, but the client will use 1.1 or 2.0, and any value put there is ignored.

So if we capture these values, the default case will be wrong any we won't be representing what the client will actually use.